cmflib/commands/metadata/push.py

```diff
@@ -87,17 +87,51 @@ def run(self):
                 response = server_interface.call_mlmd_push(json_payload, url, exec_id)
             status_code = response.status_code
             if status_code == 200 and response.json()['status']=="success":
-                return "mlmd is successfully pushed."
+                print("mlmd is successfully pushed.")
             elif status_code==200 and response.json()["status"]=="exists":
-                return "Executions already exists."
+                print("Executions already exists.")
             elif status_code==200 and response.json()["status"]=="version_update":
-                return "You need to update cmf to the latest version."
+                print("You need to update cmf to the latest version.")
```

Comment:
shouldnt we return here ? Only go ahead with tensorboard push only if metadata push is successful

Reply:
DOne